src/PDS4_NH_IngestLDD.xml

```diff
@@ -432,14 +903,40 @@
     </DD_Value_Domain>
   </DD_Attribute>
   
+  <DD_Attribute>
+    <name>spacecraft_spin_state</name>
+    <version_id>1.0</version_id>
+    <local_identifier>spacecraft_spin_state</local_identifier>
+    <nillable_flag>false</nillable_flag>
+    <submitter_name>MKC</submitter_name>
+    <definition>
+      The nh:spacecraft_spin_state attribute indicates which of the spacecraft spin states was employed during the observation sequence.</definition>
+    <DD_Value_Domain>
+      <enumeration_flag>true</enumeration_flag>
+      <value_data_type>ASCII_Short_String_Collapsed</value_data_type>
+      <DD_Permissible_Value>
+        <value>3-Axis</value>
+        <value_meaning>
+          INADEQUATE [No "TBDs" allowed. Ever.] TBD
```

Comment:
I will try to get definitions for these now, but the fact is that TBDs are sometimes unavoidable as a temporary measure. As a mission, we cannot afford (schedule-wise) to put the development of processing and archive preparation software (which depends on established structures and content of things like LDD attributes and allowable values) on hold until publication-ready definitions are produced. While it would be ideal to have fully fleshed-out documentation in place ahead of time, this is not always (maybe never?) the case and delivery deadlines would never be met if software development had to wait for all such things to be finalized.

Yes, TBDs need to be addressed prior to release. It's fairly simple to Ctrl-F "TBD"/"TBS"/"TBR" once things have settled to make sure all of our ducks are sufficiently in a row for public consumption.

It's completely fine with Kate and I if you want to hold things in a staging/fixup/development branch until ready to go, but we need to keep pushing forward on our side. I will look to see if we have anything on hand to inform a definition for these, but no promises.

Reply:
I could make something up for now, e.g. "Spinning" &rarr; "Spacecraft is spinning" and "3-Axis" &rarr; "Spacecraft pointing is triaxially stable", but the way I see it, that is potentially worse than a TBD because the intent to revisit it later is less clear and it would be much easier for something like that that we *did* want to change to end up slipping through the cracks.

Reply:
Another place where a flag (probably "[REPLACE]" for this) can be left.

Reply:
How about we use [INCOMPLETE] as our flag for definitions that we know we need, but have incomplete or inadequate items?  I like [CHECK] as the flag if we are unsure of the facts given.  That way we KNOW we need to search these two terms and ajudacate any that are remaining prior to LDD publication.